modules/EPMC.py

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env python
+
+import argparse
+import sys
+from pyspark import *
+from pyspark.sql import *
+from pyspark.sql.types import *
+from pyspark.sql.functions import *
```

Comment:
Personally I find wildcard imports confusing and also just a tad dangerous; and Python documentation [also says they should be avoided](https://docs.python.org/3/tutorial/modules.html#importing-from-a-package) (second to last paragraph)

Reply:
Actually the danger has already kicked in: `sum` imported from `pyspark.sql.functions` overwrites the base sum function.